### ISSUE TITLE
Time out connecting to ssh 

### ISSUE THREAD

**Issue by starringstars** [Strider-CD/strider-ssh-deploy#23]
Hi, 

First of all great app!

When I try to deploy i get this error. Keys are all set on my server and connection trough the terminal goes well. 

An unexpected error occurred while executing this job. Please report at https://github.com/Strider-CD/strider/issues

Host www.myhost.nl:22 -- Error: connect ETIMEDOUT

Error: Host www.myhost.nl:22 -- Error: connect ETIMEDOUT
    at Connection.<anonymous> (/Users/stars/node_modules/strider/node_modules/strider-ssh-deploy/remotely.js:68:18)
    at Connection.emit (events.js:107:17)
    at Socket.<anonymous> (/Users/stars/node_modules/strider/node_modules/strider-ssh-deploy/node_modules/ssh2/lib/Connection.js:544:10)
    at Socket.g (events.js:199:16)
    at Socket.emit (events.js:107:17)
    at net.js:451:14
    at process._tickDomainCallback (node.js:381:11)

Can you help me out what's going on ore where to find more logs. Can't find any related info in the log files nether. I've also set the readyTimeout to 999000 but still time outs.

**Comment by starringstars** [comment]
I solved this myself:) 

I think my ssh server was buggy and also the ssh-deploy because it works a bit unstable I’ve noticed. I removed the ssh-deploy plugin and reinstall this with all the keys on my ssh server and Gitlab. On my remote server there where to many processes running also but even after killing the process the plugin was acting strange sometimes. 

Finally it’s working now but I still don’t know where to place the remote path for the deployment on my ssh server. Must be an remote path variable or param somewhere but didn't found it yet. 

Very great server! Love it and hope they will keep updating this project!
